test/support/expect_issues.ex

```diff
@@ -1,4 +1,7 @@
 defmodule Nicene.ExpectIssues do
+  @moduledoc """
```

Comment:
`mix credo` pointed out that this module should have documentation 🙈

Reply:
![obraz](https://user-images.githubusercontent.com/588351/146191823-1c3222bb-228c-4b2b-b917-963df459a36f.png)